package.json

```diff
@@ -56,7 +56,7 @@
     "start": "run-s lint build",
     "bundle": "redocly bundle external@latest --output external.yml",
     "build": "run-s convert bundle redoc",
-    "postbuild": "swagger2openapi external.yml -o dist/openapi.json && openapi-typescript dist/openapi.json -o dist/index.d.ts --alphabetize --empty-objects-unknown",
```

Comment:
`--alphabetize --empty-objects-unknown` 
These flags didn't actually do anything in this version of `openapi-typescript` cli so this doesn't matter.